internal/services/bot/bot_channel_alexa_resource.go

```diff
@@ -20,6 +19,7 @@ import (
 	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
 	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
 	"github.com/hashicorp/terraform-provider-azurerm/utils"
+	"github.com/tombuildsstuff/kermit/sdk/botservice/2021-05-01-preview/botservice" // nolint: staticcheck
```

Comment:
we shouldn't need the `// nolint: staticcheck` on these btw, since the generated SDKs in Kermit aren't marked as deprecated:

```suggestion
	"github.com/tombuildsstuff/kermit/sdk/botservice/2021-05-01-preview/botservice"
```